### PR TITLE
news: add news for v0.19.0

### PR DESCRIPTION
**Pull request by ajeddeloh**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,25 @@
+22-Sep-2017 IGNITION v0.19.0
+
+  Features
+
+    - Add support for cloudstack network metadata
```
Comment (crawford) [inline NEWS]: CloudStack

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+    - Add blackbox tests for tfpt urls
```
Comment (crawford) [inline NEWS]: TFTP URLs

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+    - Removed dependency on kpartx for blackbox tests
```
Comment (bgilbert) [inline NEWS]: `Remove`

File: NEWS [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+
+  Changes
+
+    - Stop adding extra quotes around GECOS field when creating users
+    - Use a private tmpdir subdirectory hen running blackbox tests
```
Comment (bgilbert) [inline NEWS]: `when`, if this is worth release-noting at all.  It should only affect behavior when `TMPDIR` is set.

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+
+  Bug Fixes
+
+    - Fix regression in validation logic causing inaccurate line and column
+      reporting
+    - Fix regression in validation logic where json syntax errors are not
```
Comment (bgilbert) [inline NEWS]: `JSON syntax errors were`

File: NEWS [hunk 6]
```diff
@@ -1,3 +1,25 @@
...
+      reported correctly
+    - Add warnings for links and directories if a non-existent filesystem is
```
Comment (bgilbert) [inline NEWS]: `for links and directories` seems unclear.  `Add warning if non-existent filesystem is specified when creating links or directories`?

**Comment by ajeddeloh** [comment]
Nits addressed.

File: NEWS [hunk 7]
```diff
@@ -1,3 +1,24 @@
+22-Sep-2017 IGNITION v0.19.0
+
+  Features
+
+    - Add support for CloudStack network metadata
+    - Add blackbox tests for TFPT URLs
```
Comment (bgilbert) [inline NEWS]: `TFTP`

**Pull request merged**